dsc/tests/dsc_args.tests.ps1

```diff
@@ -85,7 +85,7 @@ actualState:
     ) {
         param($format, $expected)
 
-        $out = dsc --format $format resource get -r Test/Hello | Out-String
+        $out = dsc resource get -r Test/Hello --format $format | Out-String
```

Comment:
We should probably add some more tests since this parameter is now with every sub-command vs being global